README.md

```diff
@@ -66,15 +66,20 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
+* Create a kubeconfig file which has the context of all the clusters. If  KUBECONFIG environment variable set with the path to all clusters, you could use the below command.
+```
+kubectl config view --flatten > kubeconfig_merged
+```
+
 * Deploy Broker with Dataplane - To deploy the Submariner broker along with the data cluster, run using the broker-info.subm generated in the folder from which the previous step was run.
 ```
-subctl deploy-broker --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME> --clusterid  <CLUSTER-ID-FOR-TUNNELS>
+subctl deploy-broker --kubecontext <BROKER-CONTEXT-NAME>  --kubeconfig <PATH-TO-KUBECONFIG-BROKER> --dataplane --service-discovery --broker-cluster-context <BROKER-CONTEXT-NAME> --clusterid  <CLUSTER-ID-FOR-TUNNELS>
```

Comment:
Does this step require a non-merged configuration?

Reply:
Yes, else it is failing.